src/Swarm/Game/Scenario/WorldDescription.hs

```diff
@@ -52,7 +56,7 @@ instance FromJSONE (EntityMap, RobotMap) WorldDescription where
 --   string into a nested list of 'Cell' values by looking up each
 --   character in the palette, failing if any character in the raw map
 --   is not contained in the palette.
-paintMap :: MonadFail m => WorldPalette -> Text -> m [[Cell]]
+paintMap :: MonadFail m => WorldPalette Cell -> Text -> m [[Cell]]
```

Comment:
Can we make this polymorphic, `WorldPalette c -> Text -> m [[c]]` or something like that?